manifests/k/KhronosGroup/VulkanSDK/1.3.231.1/KhronosGroup.VulkanSDK.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS2.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: KhronosGroup.VulkanSDK
+PackageVersion: 1.3.231.1
+PackageLocale: en-US
+Publisher: LunarG, Inc.
```

```suggestion
Publisher: LunarG Inc.
```
This must match Control Panel / Apps&Features.